### DISCUSSION TITLE
Is there a way to display a border around shared content ?

### DISCUSSION THREAD

**Discussion by Fab06460** [MicrosoftEdge/WebView2Feedback#4418] [Q&A]
Hi,

We are developing a web collaboration application and are currently migrating from Electron to WebView2 for our desktop version.

Some time ago (I can't remember which version of WebView2), when a window was shared (via WebRTC sharing), a hightlight border was displayed around the shared window. Unfortunately I do not have a screenshot to show this.

This was very helpful because the user was well informed of what was being shared.
It seems this highlight border is no longer displayed in the latest versions of webview2.

Is there something I can configure to have it back or has this feature been removed ?

Thank you in advance,

Best regards
Fabrice.

**Comment by victorhuangwq** [comment]
@srwei do you know if this something that can be configured?
